src/appComponents/twitterwall/singletweet.js

```diff
@@ -57,7 +57,7 @@ function tweet($timeout,$rootScope) {
 
       if (scope.tweetNumber !== 90) {
             scope.tweetNumber = scope.tweetNumber + 1;
-            scope.selectedid  =scope.array[scope.tweetNumber].id_str;
+            scope.selectedid  = scope.array[scope.tweetNumber].id_str;
             $rootScope.$broadcast("selectedid",scope.selectedid);
             callchangetweet();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 space characters but found 12.](https://www.codacy.com/app/triplez/open-event-webapp/file/2539879975/issues/source?bid=3233857&fileBranchId=3284028#l62)